packages/compass-generative-ai/src/store/atlas-ai-store.ts

```diff
@@ -15,17 +20,21 @@ export function getStore() {
   }
   return store;
 }
+const reducer = combineReducers({
+  signInReducer,
+  optInReducer,
```

Comment:
nit: In Compass we'll typically name these reducers without `reducer`.
Here's a related example:
https://github.com/mongodb-js/compass/blob/28bb67513c1eca57ff14ced8d0c01562452a40a7/packages/compass-settings/src/stores/index.ts#L46

```suggestion
  signIn: signInReducer,
  optIn: optInReducer,
```